docs/PSR.md

```diff
@@ -1044,10 +1093,105 @@ This tag MUST NOT occur more than once in a "DocBlock".
 ```
 
 ### 7.13. @param
-(TODO: write up a usage description for @param)
+
+The @param tag is used to document a single parameter of a function or method.
+
+#### Syntax
+
+    @param ["Type"] [name] [<description>]
+
+#### Description
+
+With the @param tag it is possible to document the type and function of a
+single parameter of a function or method. When provided it MUST contain a
+"Type" to indicate what is expected; the description on the other hand is
+OPTIONAL yet RECOMMENDED. For complex structures such as option arrays it is
+RECOMMENDED to use an "Inline PHPDoc" to describe the option array.
+
+The @param tag MAY have a multi-line description and does not need explicit
+delimiting.
+
+It is RECOMMENDED when documenting to use this tag with every function and
+method. Exceptions to this recommendation are:
+
+This tag MUST NOT occur more than once per parameter in a "PHPDoc" and is
+limited to "Structural Elements" of type method or function.
+
+#### Examples
+
+```php
+/**
+ * Counts the number of items in the provided array.
+ *
+ * @param mixed[] $array Array structure to count the elements of.
+ *
+ * @return int Returns the number of elements.
+ */
+function count(array $items)
+{
+    <...>
+}
+```
+
+The following example demonstrates the use of an "Inline PHPDoc" to document
+an option array with 2 elements: 'required' and 'label'.
+
+```php
+/**
+ * Initializes this class with the given options.
+ *
+ * @param string[] $options {
+ *     @type boolean $required Whether this element is required
+ *     @type string  $label    The display name for this element
```

Comment:
What is the difference between `$required` and `$label`? How to distinguish mandatory one?